djmoney/contrib/exchange/models.py

```diff
@@ -22,6 +22,7 @@ def clear_rates(self):
 
 
 class Rate(models.Model):
+    id = models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name="ID")
```

Comment:
I haven't used the `serialize` option before, what does it do here?

Reply:
The intention was to write out the id field just as it is defined in the 0001 migration, and as explicitly as possible. This way we're making sure that django will not try to make any new migrations because of a difference in the field.

Reply:
The migration file defines the id field like this: `("id", models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name="ID")),`

Reply:
:100: got it, thanks :+1: